articles/tokens/guides/disable-refresh-token-rotation.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Learn how to disable Refresh Token rotation.
+topics:
+  - tokens
+  - refresh-tokens
+  - refresh-token-rotation
+contentType: how-to
+useCase:
+  - disable-refresh-token-rotation
+  - configure-refresh-token-rotation
+---
+# Disable Refresh Token Rotation
+
+Disable Refresh Token Rotation for each application using the Management API: 
```

Comment:
Same question as above - will the admin be able to disable this feature through the dashboard instead of using the API?

Reply:
Yes. Needs updated info.